build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   ws,
   specs2 % Test,
   "com.gu" %% "pan-domain-auth-play_2-4-0" % "0.2.7",
-  "com.amazonaws" % "aws-java-sdk" % "1.7.5"
+  "com.amazonaws" % "aws-java-sdk" % "1.10.50"
```

Comment:
the `AWS_SESSION_TOKEN` env var wasn't picked up in version 1.7.5 of the aws java sdk.